upgrade/upgrade-scripts/common.sh

```diff
@@ -41,10 +41,25 @@ function get_platform() {
 		sed 's/delphix-platform-//'
 }
 
-function get_installed_version() {
+function get_installed_entire_version() {
 	dpkg-query -Wf '${Version}' "delphix-entire-$(get_platform)"
 }
 
+#
+# The virtualization package may not always exist (i.e. internal minimal
+# builds). In that case, we fall back to the delphix-entire version as the
+# product version.
+#
+function get_installed_product_version() {
```

Comment:
If "the product" isn't installed, would it be better to return an empty string from this function? This way, it's more clear that "the product" isn't installed, rather than trying to hide that fact from the user..?

To me, if "the product" is the virtualization package, and that package isn't installed, it seems wrong to lie and return the version of the delphix-entire package.. even if this behavior makes sense based on how this function is used, from an API/interface standpoint, this behavior strikes me as awkward.